src/modules/tools/services/googleImages.js

```diff
@@ -0,0 +1,88 @@
+'use strict';
+
+const Got = require('got');
+
+// eslint-disable-next-line no-unused-vars
+const { MessageEmbed } = require('discord.js');
+
+const { Service } = require('../../../core');
+
+/**
+ * @typedef {Object} ResultObject
+ *
+ * @property {String} title
+ * @property {String} link
+ * @property {String} displayLink
+ * @property {ResultImageObject} image
+ */
+
+/**
+ * @typedef {Object} ResultImageObject
+ *
+ * @property {String} contextLink
+ * @property {Number} width
+ * @property {Number} height
+ * @property {Number} byteSize
+ * @property {String} thumbnailLink
+ * @property {Number} thumbnailWidth
+ * @property {Number} thumbnailHeight
+ */
+
+module.exports = class GoogleImagesService extends Service {
+
+    #api;
+
+    init() {
+        this.#api = Got.extend({
+            prefixUrl: 'https://www.googleapis.com',
+        });
```

Comment:
Why not set the options `json` to `true` to avoid JSON parsing on line 61 ?

Reply:
For some odd reason, the request fails with 400. In general, the CSE API seems to be really finicky like that, returning 400 with barely any info on anything slightly out of order.

Reply:
Alright, freaking google making weird API I guess